core/src/trezor/wire/thp/interface_context.py

```
@@ -54,186 +45,422 @@ class ThpContext:
+                    f"Interrupted channel {hex(self.active_channel.channel_id)} after {last_write_ms} ms",
+                )
+            self.active_channel.kill(ChannelPreemptedException())
+
+    async def close(self) -> None:
+        try:
+            close_tasks = [
+                loop.spawn(iface_ctx.close()) for iface_ctx in self._iface_ctxs
+            ]
+            for task in close_tasks:
+                await task
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)